src/services/Store/StoreProvider.tsx

```diff
@@ -197,7 +196,6 @@ export const StoreProvider: React.FC = ({ children }) => {
           isSameAddress(senderAccount.address, acc.address)
         ) as StoreAccount;
 
-        const txs = getTxsFromAccount([storeAccount]);
```

Comment:
Your change here does not do the same as the previous code. You need to filter based on the account in question, otherwise we will get a match for transaction that is not overwriting the pending one.

Reply:
Reverted.